tests/fixtures/fixtures.py

```diff
@@ -164,8 +164,20 @@ def _session_tiny_bert_config():  # type: ignore
     return hf_config
 
 
+def tiny_gpt_model_helper(config):
+    transformers = pytest.importorskip('transformers')
+
+    hf_model = transformers.AutoModelForCausalLM.from_config(config)
+    return hf_model
+
+
 @pytest.fixture(scope='session')
-def _session_tiny_gpt2_config():  # type: ignore
+def _session_tiny_gpt_model(_session_tiny_gpt_config):  # type: ignore
+    hf_model = tiny_gpt_model_helper(_session_tiny_gpt_config)
+    return hf_model
```

Comment:
```suggestion
    return tiny_gpt_model_helper(_session_tiny_gpt_config)
```
what's the point of splitting this function out? can we flatten

Reply:
yeah, sure. no reason